### DISCUSSION TITLE
Is it possible to connect 2 folders for downloads?

### DISCUSSION THREAD

**Discussion by evil-pechenka** [haugene/docker-transmission-openvpn#2300] [Q&A]
Is it possible to connect a second folder for downloads? For example, I have 2 discs and want to download to both?

I mount the first one in / data, but is it possible to mount the second one?

**Comment by pkishino** [comment]
you can mount as many volumes as you want, just specify where they should go in the container and then use that location in transmission

**Reply by evil-pechenka** [reply]
Thank you! I mounted **/disk2/folder2 : /data1** and everything downloads fine.

In fact, I was confused by this text in the instructions:

"**1. The container assumes that you mount a folder to /data**
Technically you don't have to do this, but it is by far the most manageable way of getting the downloaded files onto your host system and Transmission will store it's state there. So if you don't mount this directory then you will loose all your torrents on image updates."
(https://haugene.github.io/docker-transmission-openvpn/run-container/)

Will there be problems with torrents with updates?
